app/models/user.rb

```diff
@@ -1,3 +1,16 @@
-# rubocop:disable all
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
+  has_many :posts, class_name: 'Post', foreign_key: 'author_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'author_id'
+  has_many :comments, class_name: 'Comment', foreign_key: 'author_id'
+
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  def recent_posts
+    Post.order(created_at: :desc).limit(3)
+  end
 end
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** What do you think if you create a `user.admin?` method inside the user model to make your code more professional. 
```ruby
 # Example
   ...
   def admin?
     role == 'admin' ? true : false
   end
```